### ISSUE TITLE
NPM version 0.0.8 not up to date

### ISSUE THREAD

**Issue by rdarcy1** [baianat/hooper#44]
The NPM package version 0.0.8 does not appear to be up to date with the source tagged as 0.0.8 on GitHub. 

I noticed that the `wheelControl` property defaults to `true` whereas it was set to `false` in the latest version. Not sure if there are any other differences.

See https://unpkg.com/hooper@0.0.8/dist/hooper.js

**Comment by Mucaccino** [comment]
Hi,

To confirm that issue, here the same, the package is 0.0.8 but the dist sources being 0.0.7 without new changes, like referenced by @rdarcy1 upside.

![image](https://user-images.githubusercontent.com/399292/53030828-63e9c600-344a-11e9-8e71-0795676041aa.png)

**package.json:**

```
{
  ...
  "unpkg": "dist/hooper.js",
  "version": "0.0.8"
}
```


**dist/hoper.js:**

Version:

```
/**
  * Hopper 0.0.7
  * (c) 2019
  * @license MIT
  */
```

WhelControl default keep true instead new false:

```
 // toggle mouse wheel sliding
 wheelControl: {
   default: true,
   type: Boolean
 },
```

Hugs,

**Comment by Abdelrahman3D** [comment]
This should be fixed in the new version `0.0.9`

**Comment by rdarcy1** [comment]
Thanks!
